.vscode/launch.json

```diff
@@ -20,7 +20,9 @@
                     "text": "-enable-pretty-printing",
                     "ignoreFailures": true
                 }
-            ]
+            ],
+            "preLaunchTask": "Meson: Build all targets"
```

Comment:
Is there a reason why you add here the attribute `preLaunchTask`?

Reply:
Because it is usually better to build an executable before trying to launch it. ;)

Reply:
... ahh, right.